base/deprecated.jl

```diff
@@ -1763,4 +1763,13 @@ end)
     return false
 end
 
+# LibGit2.owner -> LibGit2.repository for consistency
+eval(Base.LibGit2, :(Base.@deprecate_binding owner repository))
```

Comment:
don't need the deprecate_binding (which will still export) if you define the method below